tfhe/benches/core_crypto/pbs_bench.rs

```diff
@@ -92,7 +62,51 @@ fn benchmark_parameters<Scalar: Numeric>() -> Vec<(String, BenchmarkPbsParameter
     }
 }
 
+fn multi_bit_benchmark_parameters<Scalar: Numeric>(
+) -> Vec<(String, (CryptoParametersRecord, LweBskGroupingFactor))> {
+    if Scalar::BITS == 64 {
+        vec![
+            (
+                "4_bits_multi_bit_group_2".to_string(),
```

Comment:
still unsure about this, do you want to add the multi bit in the same PR ?

Reply:
Yes we'll keep your initial commit so we can get two birds with one stone.